config/env_spec.go

```diff
@@ -225,6 +225,8 @@ type APISpec struct {
 	// ID of the API, used to match the api_source of API Product Operations.
 	ID string `yaml:"id" mapstructure:"id"`
 
+	RevisionID string `yaml:"revision_id" mapstructure:"revision_id,omitempty"`
```

Comment:
Please add comments with `// RevisionID ...`

Reply:
Also remove the omitempty in the mapstructure tag.

Reply:
Done.

Reply:
nit: please remove the omitempty part

Reply:
Done.